js/main.js

```diff
@@ -5,10 +5,12 @@ var $img = document.querySelector('div.column-half>img');
 var $form = document.querySelector('#journal-entry');
 var $dataViewDivs = document.querySelectorAll('div[data-view]');
 var ul = document.querySelector('ul.padding-left-0');
+var EntryFormHeader = document.querySelector('form#journal-entry>div.column-full>h1');
```

Comment:
Variable names should never start with a capital letter, that has a special meaning that we will discuss later. Please change this to `camelCase`